WordPress/Classes/ViewRelated/Blog/Site Picker/SiteSwitcherReducer.swift

```diff
@@ -0,0 +1,18 @@
+enum SiteSwitcherReducer {
```

Comment:
I don't think this should be an enum as well.

Reply:
This one has no state though. Why would we want to have an initialisable type here?

Reply:
For readability. We can have a final class with static functions that will serve the same purpose. But having an enum implies it holds enumerations, either cases or constants, not logic.

Reply:
How does having this as class or struct enhance readability? From the caller's perspective they are indifferent apart from the visible type in the autocomplete. Also, we have tons of `enum Constants` for the very same reason. If we only need namespacing and not going to hold any state, an enum ensures that can't happen and cannot be constructed. Afaik even Apple uses enum as namespacing instead of struct or class in frameworks like combine

Reply:
> Afaik even Apple uses enum as namespacing instead of struct or class in frameworks like combine

Interesting, can you share a link to that?

Reply:
I can't find their exact implementation but asking chatgpt (of course should be taken with a grain of salt) also supports this.

I asked:
> Does Apple use struct or enum for namespacing in their Swift code?

ChatGPT's answer:
> Apple leans towards using enum for namespacing when there's no need to create instances of the namespace. This approach is particularly evident in cases where the namespace groups constants, utility functions, or related types that do not need to maintain state. Since enums cannot be instantiated, using an enum for this purpose makes it clear that the namespace is meant solely for organizing code.

Reply:
> the namespace groups constants, utility functions, or related types that do not need to maintain state.

This does make sense, and I personally use it for constants and related types. I haven't used it before for utility functions, and I assume this is the case for this enum. If so, can we rename the enum since the current name implies that it's a type that does something (reduce), not a collection of utility functions?

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/22833/commits/0c91c5df027ca82901f837cdb08f31b8792570ee